pkg/utils/jsonutils_test.go

```diff
@@ -279,3 +279,18 @@ func TestFlatten(t *testing.T) {
 		})
 	}
 }
+
+func TestGetString_BufferResizing(t *testing.T) {
+    data := []byte(`{"key": "value"}`)
+    var workBuf []byte // Initialize an empty buffer
+
+    result, err := GetString(data, workBuf, "key")
+    if err != nil {
+        t.Fatalf("expected no error, got %v", err)
```

Comment:
don't use Fatal but rather asserts, checkout the rest of the test functions how they assert test outputs

Reply:
updated